models/yang/sonic/sonic-nat.yang

```diff
@@ -0,0 +1,218 @@
+module sonic-nat {
+    namespace "http://github.com/Azure/sonic-nat";
+    prefix snat;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-common {
+        prefix scommon;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC NAT";
+
+    revision 2019-10-30 {
+        description
+            "Initial revision.";
+    }
+
+
+    container sonic-nat {
+
+        container STATIC_NAPT {
+
+            list STATIC_NAPT_LIST {
+                key "global_ip ip_protocol global_l4_port";
```

Comment:
Please check if we should use max-elements to limit the number of list instances. Check for all list below as well.

Reply:
Addressed